vizro-core/docs/pages/development/documentation-style-guide.md

```diff
@@ -0,0 +1,143 @@
+# Vizro documentation style guide
+
+This is the style guide we apply to the [Vizro documentation](https://vizro.readthedocs.io/en/stable/).
+
+We ask anyone kind enough to contribute documentation changes to follow this style for consistency and simplicity.
+
+What follows is a set of lightweight guidelines rather than rules. There are always edge cases and exceptions, and if it's not obvious what the style should be, consult the [Microsoft style guide](https://docs.microsoft.com/en-gb/style-guide/welcome/) for an example of good practice. We also use the [INCITS Inclusive Terminology Guidelines](https://standards.incits.org/apps/group_public/download.php/131246/eb-2021-00288-001-INCITS-Inclusive-Terminology-Guidelines.pdf).
+
+## Vizro lexicon
+
+The names of our products are **Vizro** and **Vizro-AI**.
+
+We refer to other products using their preferred capitalization. For example, Dash and Plotly are always capitalized, except where given as Python package names `dash` and `plotly`.
+
+Vizro components are named using lower case:
+
+> "Here is a guide to using containers..."
```

Comment:
```suggestion
> Here is a guide to using containers...
```

(comment applies in a few places)

Unless you meant these to be here for some particular reason? They seem to be in some quotes but not others.